ft_prepare_mesh.m

```diff
@@ -16,7 +16,8 @@
 %
 % Configuration options:
 %   cfg.method      = string, can be 'interactive', 'projectmesh', 'iso2mesh', 'isosurface',
-%                     'headshape', 'hexahedral', 'tetrahedral', 'cortexhull'
+%                     'headshape', 'hexahedral', 'tetrahedral',
+%                     'cortexhull', 'fittemplate'
```

Comment:
please keep them on a single line (i.e. 2 in total, not 3)

Reply:
Put it back to 2 lines